From 8ec397c7c770cd257b11c16fbaa3094a357d3cb3 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 1 Jan 2018 21:23:00 +0100
Subject: [PATCH 1/1] fs/fat: remove distractive message in file_fat_read_at()

The message "reading %s\n" may be interesting when
debugging but otherwise it is superfluous.

Only output the message when debugging.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 fs/fat/fat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/fat/fat.c b/fs/fat/fat.c
index d16883fa10..1283818761 100644
--- a/fs/fat/fat.c
+++ b/fs/fat/fat.c
@@ -1106,7 +1106,7 @@ int file_fat_read_at(const char *filename, loff_t pos, void *buffer,
 	if (ret)
 		goto out_free_both;
 
-	printf("reading %s\n", filename);
+	debug("reading %s\n", filename);
 	ret = get_contents(&fsdata, itr->dent, pos, buffer, maxsize, actread);
 
 out_free_both:
-- 
2.14.2

